go-controller/pkg/kubevirt/switch.go

```diff
@@ -53,3 +63,39 @@ func ComposeARPProxyLSPOption() string {
 	}
 	return strings.Join(arpProxy, " ")
 }
+
+// notifyARPProxyMACForIP will send an GARP to force arp caches to clean up
+// at pods and reference to the arp proxy gw mac
+func notifyARPProxyMACForIPs(ips []*net.IPNet, dstMAC net.HardwareAddr) error {
```

Comment:
This function is one of the core points from this PR. I know you described the global picture in the markdown document, but could you also describe here what's happening in the relevant functions and in the commit message?

Reply:
Hi, so the commits implementing the design doc are from the PoC, if implementatino is "sound" I will refactor them with proper commit messages and structure.